src/python/elements.cpp

```diff
@@ -669,13 +684,17 @@ void init_elements(py::module& m)
                 amrex::ParticleReal,
                 amrex::ParticleReal,
                 amrex::ParticleReal,
+                amrex::ParticleReal,
+                amrex::ParticleReal,
                 int,
                 std::optional<std::string>
              >(),
              py::arg("ds"),
              py::arg("dx") = 0,
              py::arg("dy") = 0,
              py::arg("rotation") = 0,
+             py::arg("xmax") = 0,
+             py::arg("ymax") = 0,
```

Comment:
If we use `std::optional` then this changes to
```suggestion
             py::arg("xmax") = py::none(),
             py::arg("ymax") = py::none(),
```